assets/global.js

```diff
@@ -599,7 +599,10 @@ class VariantSelects extends HTMLElement {
     if(this.stickyHeader) {
       this.stickyHeader.dispatchEvent(new Event('preventHeaderReveal'));
     }
-    window.setTimeout(() => { parent.querySelector('li.product__media-item').scrollIntoView({behavior: "smooth"}); });
+    window.setTimeout(() => {
+      parent.scrollLeft = 0;
+      parent.querySelector('li.product__media-item').scrollIntoView({behavior: "smooth"});
```

Comment:
I think removing the `{behavior: "smooth"}` would also fix this issue. I feel this might have to do with limited support of the `smooth` behaviour option. Some browsers may just ignore the option, but others (older versions of Android) may not initiate the scroll altogether.

![Screen Shot 2021-10-19 at 4 38 56 PM](https://user-images.githubusercontent.com/3710548/137987177-82cc0c10-9756-4932-9431-c4bab8145119.png)